misc/pp_mixed_scenarios/README.md

```diff
@@ -0,0 +1,65 @@
+# Post-Processing with Mixed Scenarios
+
+This script is used for post-processing results generated from mixed scenarios. It is essential that
+the directory structure is properly set up for the script to function correctly. Please follow the
```

Comment:
🚫 Can we link or include installation instructions. Presumably it is the same as the repo itself (i.e. they need poetry, and then run `poetry install`)?